src/test/java/io/github/ust/mico/kafkafaasconnector/MessageListenerTests.java

```diff
@@ -77,6 +84,15 @@
     @Autowired
     MessageListener messageListener;
 
+    @Autowired
+    FaasController faasController;
```

Comment:
Proposal: Extract the unit tests that only tests one specific service to its own test class:
- `FaasControllerTests`
- `CloudEventManipulatorTests` 
- `KafkaMessageSenderTests`
- `CloudEventDeserializerTests`